packages/providers/base/assets/Provider.ts

```diff
@@ -1,31 +1,70 @@
-import { SNS, SSM } from 'aws-sdk';
+import { SNS, SSM, Credentials, CredentialProviderChain } from 'aws-sdk';
 import { Logger } from './Logger';
 
+interface Account {
+    accountId: string;
+    credentials: string;
+}
+
 export class Provider {
     public deploymentGuid: string;
     public environment: string;
     public componentName: string;
+	public config: Array<{ Key: string, Value: string }>;
     public inputs: Array<{ Key: string, Value: string }>;
     public outputs: Array<{ Key: string, Value: string }>;
     public result: boolean;
     public jobRunGuid: string;
     public logGroup: string;
     public logger: Logger;
+    public account: Account;
+
+	public awsCredentials: Credentials;
+	public region: string;
+
     sns = new SNS();
     ssm = new SSM();
 
     constructor(event: any) {
+        console.log(event);
```

Comment:
We can't necessary log the event here because some providers could end up logging sensitive data.

Reply:
I removed this log line.